debian/bullseye/foreman-release/foreman.list

```diff
@@ -0,0 +1,3 @@
+deb http://deb.theforeman.org/ bullseye 3.4
+deb http://deb.theforeman.org/ plugins 3.4
```

Comment:
Shouldn't this be `nightly` now?

Reply:
ah sorry, i tried the sed locally and ended up pushing the 3.4